### DISCUSSION TITLE
Notion Sync Problem

### DISCUSSION THREAD

**Discussion by 0xBentz1** [morethanmin/morethan-log#278] [Q&A]
Hey, 
When a i'm adding a new Notion post, it is not updated automatically. i need to redeploy the project in Vercel for it to be updated.
What can cause the problem? thanks

**Comment by Seung-zedd** [comment]
hello i was about to ask your similar problem, but hopefully i found your question. In my shoes, i fixed synchronization problem by querying into URL in the first-hand manner. At first, it shows 404 error, but after a few minute, your uploaded posts was shown in the browser
Hope you fix your problem as well!

**Reply by lucas-nobrega** [reply]
How did you do it?

**Comment by seopchan** [comment]
#224 
The `revalidate` value in `src/pages/[slug].tsx` has been changed from 1 to `CONFIG.revalidateTime` (21600 * 7). 
You can verify the desired automatic update functionality by using `src/pages/api/revalidate.ts` or by reducing the `CONFIG.revalidateTime` duration.

**Reply by seungjun-Lee0** [reply]
Thanks for your help :) I have something to ask.
Can I just set revalidateTime to 1 same as previous value? and do you know why it has been changed?

**Reply by seopchan** [reply]
Change the 'revalidateTime' in 'site.config.js' to the desired time. 
For the reason behind the change, refer to https://github.com/morethanmin/morethan-log/pull/224#discussion_r1249447371
